Sources/Event.php

```diff
@@ -1115,7 +1115,7 @@ protected function getNumDays(): int
 			return 0;
 		}
 
-		return date_interval_format(date_diff($this->start, $this->end), '%a') + ($this->end->format('H') < $this->start->format('H') ? 2 : 1);
+		return ((int) $this->start->diff($this->end)->format('%a')) + ($this->end->format('H') < $this->start->format('H') ? 2 : 1);
```

Comment:
I refactored this when I came across it because my IDE didn't like the date_diff returns a DateInterval and passed that into date_interval_format, which expects param 1 to b ea string.  A bit of reworking and its doing what it needs to do, type-safe.